setup.cfg

```diff
@@ -1,127 +0,0 @@
-# https://github.com/pyscaffold/pyscaffold/blob/master/setup.cfg
-
-[metadata]
-name = ga4gh.vrs
-home_page = https://github.com/ga4gh/vrs-python
-long_description = file:README.md
-long_description_content_type = text/markdown; charset=UTF-8
-description = "GA4GH Variation Representation Specification (VRS) reference implementation (https://github.com/ga4gh/vrs-python/)"
-#description-content-type = text/x-rst; charset=UTF-8
-license_files = LICENSE
-author = Reece Hart
-author_email = reecehart@gmail.com
-platform = any
-# version is pulled from tags by setuptools_scm
-
-classifiers =
-    Development Status :: 3 - Alpha
-    Intended Audience :: Developers
-    Intended Audience :: Healthcare Industry
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: Apache Software License
-    Operating System :: OS Independent
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
-    Topic :: Scientific/Engineering :: Bio-Informatics
-    Topic :: Scientific/Engineering :: Medical Science Apps.
-keywords =
-    bioinformatics
-    ga4gh
-    genomics
-    hgvs
-    spdi
-    variation
-
-
-[options]
-packages = find_namespace:
-include_package_data = True
-python_requires = >= 3.8
-zip_safe = True
-package_dir =
-    = src
-install_requires =
-    bioutils
-    canonicaljson
-    coloredlogs
-    numpy
-    pyyaml
-    pydantic~=2.1
-setup_requires =
-    cython
-    pytest-runner
-    setuptools_scm
-    wheel
-tests_require =
-    pytest
-    pytest-cov
-    pytest-vcr
-    vcrpy
-
-[options.packages.find]
-where = src
-exclude =
-    __pycache__
-    *.pyc
-
-
-# Jedi pinned https://github.com/ipython/ipython/issues/12677#issuecomment-752258403
-[options.extras_require]
-dev =
-    ipython
-    jedi<0.18
-    pylint
-    pytest
-    pytest-cov
-    pytest-vcr
-    pyyaml
-    restview
-    smart_open
-    sphinx
-    sphinx_rtd_theme
-    vcrpy
-    yapf
-extras =
-    psycopg2-binary
-    biocommons.seqrepo>=0.5.1
-    bioutils>=0.5.2
-    hgvs>=1.4
-    requests
-    dill~=0.3.7
-    click
-notebooks =
-    ipython
-    jupyter
-    tabulate
-
-
-[aliases]
-test = pytest
-
-[bdist_wheel]
-universal = 1
-
-[build_sphinx]
-all_files  = 1
-
-# http://pep8.readthedocs.org/en/latest/intro.html#error-codes
-[flake8]
-max-line-length = 120
-exclude = tests/*
-max-complexity = 10
-ignore = E129,E221,E241,E251,E303,W291
-
-[tool:pytest]
-# TODO: add src to testpaths to trigger doctests. However, this
-# requires injecting fixtures, which requires conftest.py in src/,
-# which requires packaging changes. Deferred.
-testpaths = src tests
-doctest_optionflags = ALLOW_UNICODE ALLOW_BYTES ELLIPSIS IGNORE_EXCEPTION_DETAIL NORMALIZE_WHITESPACE
-addopts = --cov-config=setup.cfg --cov-report=term-missing --cov=ga4gh
```

Comment:
`setup.cfg` is given as the pytest-cov config target but no actual configuration options are set. I removed it.